src/main/java/org/traccar/model/ExtendedModel.java

```diff
@@ -91,7 +91,8 @@ public void add(Map.Entry<String, Object> entry) {
 
     public String getString(String key, String defaultValue) {
         if (attributes.containsKey(key)) {
-            return attributes.get(key).toString();
+            Object value = attributes.containsKey(key);
```

Comment:
Wait. There's a typo. It should obviously be `.get(key)` instead of `.containsKey(key)`

Reply:
My bad. Fixed.